config/src/config/consensus_config.rs

```diff
@@ -176,30 +176,30 @@ impl Default for ConsensusConfig {
                 ChainHealthBackoffValues {
                     backoff_if_below_participating_voting_power_percentage: 80,
                     max_sending_block_txns_override: 2000,
-                    max_sending_block_bytes_override: 500 * 1024,
+                    max_sending_block_bytes_override: 1024 * 1024,
                 },
                 ChainHealthBackoffValues {
                     backoff_if_below_participating_voting_power_percentage: 77,
                     max_sending_block_txns_override: 1000,
-                    max_sending_block_bytes_override: 250 * 1024,
+                    max_sending_block_bytes_override: 1024 * 1024,
                 },
                 ChainHealthBackoffValues {
                     backoff_if_below_participating_voting_power_percentage: 75,
                     max_sending_block_txns_override: 400,
-                    max_sending_block_bytes_override: 100 * 1024,
+                    max_sending_block_bytes_override: 1024 * 1024,
                 },
                 ChainHealthBackoffValues {
                     backoff_if_below_participating_voting_power_percentage: 72,
                     max_sending_block_txns_override: 200,
                     // stop reducing size, so 100k transactions can still go through
-                    max_sending_block_bytes_override: 100 * 1024,
+                    max_sending_block_bytes_override: 1024 * 1024,
                 },
                 ChainHealthBackoffValues {
                     backoff_if_below_participating_voting_power_percentage: 69,
                     // in practice, latencies make it such that 2-4 blocks/s is max,
                     // meaning that most aggressively we limit to ~200-400 TPS
                     max_sending_block_txns_override: 100,
```

Comment:
if batch size of 100 is too small, we can remove this level. What do you think?

(I can add delay to chain health backoff as well, and then we shouldn't need so small blocks)

Reply:
Bigger is probably better 😄 but do you think it is safe to remove it without additional testing?